From fdd05bcbf2b0f037daf0b1178c0538350e4cd7b6 Mon Sep 17 00:00:00 2001
From: Felipe Borges <felipeborges@gnome.org>
Date: Wed, 25 Mar 2020 16:49:43 +0100
Subject: [PATCH] GVirDomain: Add GVIR_DOMAIN_DELETE_{REMOVE,KEEP}_NVRAM

These are mapped respectively to VIR_DOMAIN_UNDEFINE_NVRAM and
VIR_DOMAIN_UNDEFINE_KEEP_NVRAM.

Fixes issues such as https://bugzilla.redhat.com/1817031

Signed-off-by: Felipe Borges <feborges@redhat.com>
---
 libvirt-gobject/libvirt-gobject-domain.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libvirt-gobject/libvirt-gobject-domain.h b/libvirt-gobject/libvirt-gobject-domain.h
index 099cde3..8e55d17 100644
--- a/libvirt-gobject/libvirt-gobject-domain.h
+++ b/libvirt-gobject/libvirt-gobject-domain.h
@@ -104,6 +104,8 @@ typedef enum {
     GVIR_DOMAIN_DELETE_NONE               = 0,
     GVIR_DOMAIN_DELETE_SAVED_STATE        = VIR_DOMAIN_UNDEFINE_MANAGED_SAVE,
     GVIR_DOMAIN_DELETE_SNAPSHOTS_METADATA = VIR_DOMAIN_UNDEFINE_SNAPSHOTS_METADATA,
+    GVIR_DOMAIN_DELETE_REMOVE_NVRAM       = VIR_DOMAIN_UNDEFINE_NVRAM,
+    GVIR_DOMAIN_DELETE_KEEP_NVRAM         = VIR_DOMAIN_UNDEFINE_KEEP_NVRAM,
 } GVirDomainDeleteFlags;
 
 /**
-- 
2.26.2

